ospf6d/ospf6_memory.h

```diff
@@ -43,5 +43,6 @@ DECLARE_MTYPE(OSPF6_EXTERNAL_INFO)
 DECLARE_MTYPE(OSPF6_PATH)
 DECLARE_MTYPE(OSPF6_DIST_ARGS)
 DECLARE_MTYPE(OSPF6_OTHER)
+DECLARE_MTYPE(OSPF6_REDISTRIBUTE)
```

Comment:
Just a suggestion: It is better to keep DECLARE_MTYPE(OSPF6_OTHER) at the end since it logically kind of marks the end of types.

Reply:
Fixed.